### PR TITLE
ArraysUtils added

### PR DESCRIPTION
**Pull request by bataborisa**
I have added some ArraysUtil methods

### CODE REVIEW

**Review by emiryazicii — COMMENTED** [review]
Thanks Boris, It looks great. We will discuss what methods we can write in that class.
I just want to remind you , it will be perfect if you can make a pull request to 'develop' branch instead 'master' branch directly.

File: src/utils/ArraysUtils.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package utils;
+
+public class ArraysUtils {
+
+    // methods we need
```
Comment (emiryazicii) [inline src/utils/ArraysUtils.java:5]: That will be great add some methods here later on.For now it looks good enough.

**Comment by emiryazicii** [comment]
I just want to remind you That will be perfect if you can make a pull request to develop branch instead master branch directly.

**Review by emiryazicii — CHANGES_REQUESTED** [review]
Can you please send the pull request to "develop" branch.

**Comment by bataborisa** [comment]
I did change mine pull request, so now is from my name Branch to Devalop branch, hope now everything will be at right place

**Comment by emiryazicii** [comment]
Thanks for update.

**Pull request closed without merging**